test/runtests.jl

```diff
@@ -1446,3 +1564,148 @@ end
+                      0.010149793505874801 0.010149793505874801 0.0 0.0 0.010149793505874801; 
+                     -1.8788313148033928 -0.0 -1.8788313148033928 -0.0 -1.8788313148033928]
+        
+        
+        
+        @test mm0_pois ≈  GLM.momentmatrix(gm_pois) atol=1e-06
+        @test mm0_poisw ≈  GLM.momentmatrix(gm_poisw) atol=1e-06
+    end
+    @testset "Binomial" begin
+        f = @formula(admit ~ 1 + rank)
+        
+        gm_bin = fit(GeneralizedLinearModel, f, admit_agr, Binomial())
+        mm0_bin = [-0.5  -0.0  -0.0  -0.0
+                   -0.5  -0.5  -0.0  -0.0
+                   -0.5  -0.0  -0.5  -0.0
+                   -0.5  -0.0  -0.0  -0.5
+                    0.5   0.0   0.0   0.0
+                    0.5   0.5   0.0   0.0
+                    0.5   0.0   0.5   0.0
+                    0.5   0.0   0.0   0.5]        
+        @test mm0_bin ≈ GLM.momentmatrix(gm_bin)     
+
+        gm_binw = fit(GeneralizedLinearModel, f, admit_agr, Binomial(), wts=aweights(admit_agr.count))
+        mm0_binw =  [-15.1475    -0.0      -0.0     -0.0
+                     -34.6887   -34.6887   -0.0     -0.0
+                     -21.5207    -0.0     -21.5207  -0.0
+                      -9.85075   -0.0      -0.0     -9.85075
+                      15.1475     0.0       0.0      0.0
+                      34.6887    34.6887    0.0      0.0
+                      21.5207     0.0      21.5207   0.0
+                       9.85075    0.0       0.0      9.85075]
+        
+        @test mm0_binw ≈ GLM.momentmatrix(gm_binw) atol=1e-03
+        Vcov =[ 0.0660173  -0.0660173  -0.0660173  -0.0660173
+               -0.0660173   0.0948451   0.0660173   0.0660173
+               -0.0660173   0.0660173   0.112484    0.0660173
+               -0.0660173   0.0660173   0.0660173   0.167532]
+
+        ## This is due to divverences between chol and qr
+        @test vcov(gm_binw) ≈ Vcov atol=1e-03
+
+        gm_binw = fit(GeneralizedLinearModel, f, admit_agr, Binomial(), wts=pweights(admit_agr.count))
+        @test mm0_binw ≈ GLM.momentmatrix(gm_binw) atol=1e-03
+        ## This are obtained from stata
+        ## glm admit i.rank [pweight=count], family(binomial)  irls
+        coef_stata = [.16430305, -.75002998, -1.364698,  -1.6867296]
+        @test coef(gm_binw) ≈ coef_stata atol=1e-05
+        ## Stata: uses different residuals degrees of freedom. In this case (n-1) instead of (n-4)
+        ## Also need to give low tolerance (this small differences seem to be due to QR vs Cholesky)
+        @test stderror(gm_binw)*sqrt(5/7) ≈ [1.5118579, 2.1380899, 2.1380899, 2.1380899] atol=1e-02
+
+        ## Stata is also off with fweights
+        gm_binw = fit(GeneralizedLinearModel, f, admit_agr, Binomial(), wts=fweights(admit_agr.count))
+        ## vs Stata (here stata uses the same df)
+        stata_se = [.25693835, .30796933,  .33538667,   .4093073]  
+        @test stderror(gm_binw) ≈  stata_se atol = 0.001
```

Comment:
We're supposed to test `momentmatrix` here, not `coef`. Put this in analytic_weights.jl, etc.?

Also test `momentmatrix` (and `modelmatrix`) with `weighted=true`?